### ISSUE TITLE
TestMutationResizeMemoryUp, pod is not ready yet

### ISSUE THREAD

**Issue by artemnikitin** [elastic/cloud-on-k8s#1088]
```go
11:03:22      --- FAIL: TestMutationResizeMemoryUp/Pods_should_eventually_be_ready (300.00s)
11:03:22          require.go:794: 
11:03:22              	Error Trace:	testutils.go:42
11:03:22              	Error:      	Received unexpected error:
11:03:22              	            	Pod test-mutation-resize-memory-up-es-svmmwvt9sv is not ready yet
11:03:22              	Test:       	TestMutationResizeMemoryUp/Pods_should_eventually_be_ready
11:03:22  FAIL
```

**Comment by artemnikitin** [comment]
Most common issue for recent test failures, feels like 80-90% of all recent failures

**Comment by barkbay** [comment]
Should be fixed by https://github.com/elastic/cloud-on-k8s/pull/1083/commits/ab5dde6290756cb52e5eadaf2453bb0e8cea4cd4#diff-16b5d8fdde374517d3867df8ad9e6f75 when I will be able to merge

**Comment by barkbay** [comment]
fixed in #1083
